x-pack/plugins/graph/public/components/search_bar.tsx

```diff
@@ -161,6 +172,17 @@ export function SearchBarComponent(props: SearchBarStateProps & SearchBarProps)
             })}
             query={query}
             onChange={setQuery}
+            appName={appName}
+            deps={{
+              unifiedSearch,
+              usageCollection,
```

Comment:
Looks like `usageCollection` is undefined here. Is it expected?

Reply:
Thanks for the comment. No, it should work. Could you tell me how you reproduced it? which page did you go to?

Reply:
Analytics > Graph (when elasticsearch is started with ` --license trial`) 

<img width="1497" alt="Screenshot 2022-09-19 at 19 27 37" src="https://user-images.githubusercontent.com/1415710/191077744-d2bc7766-e223-49bf-b9a8-acc2d39b87d1.png">